server/BraveAlerterConfigurator.js

```diff
@@ -99,6 +100,20 @@ class BraveAlerterConfigurator {
 
         // If the SMS came from the session's respondedByPhoneNumber
         if (alertSession.respondedByPhoneNumber === session.respondedByPhoneNumber) {
+          // If the client sends a request to reset, reject it by not updating the session as stored in the database
+          // NOTE: this *should* never happen, as the getClientMessageForRequestToReset function returns null
+          // Thus, the alert state machine should never transition from `STARTED` or `WAITING_FOR_REPLY` to `RESET`
+          if (alertSession.alertState === CHATBOT_STATE.RESET) {
+            await db.rollbackTransaction(pgClient)
+
+            // Don't send messages to anyone; an error on part of Brave or Twilio likely caused the `RESET` state to be entered
```

Comment:
Hmmm, interesting situation. 

I'm not sure that you do want to return anything special in this case. If you were to just `return { respondedByPhoneNumber: session.respondedByPhoneNumber }` at the bottom of the function like all the other paths, would it send the "Thank you" text message? Maybe that's OK. It's what we do if there were any other errors, right?

Reply:
Actually, come to think of it, the extra return values here are extraneous as the getReturnMessageToX returns null for any RESET state transition. Do you think it should have actual messages?